src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```
@@ -175,7 +183,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       Overwrite="true" />
     <!-- Requires Windows 10 version 1803 or newer -->
     <Exec
-      Command="tar -czf $(TarArchiveOutputPath) ."
+      Command="$(TarCommand) -czf $(TarArchiveOutputPath) ."
```

```suggestion
      Command="&quot;$(TarCommand)&quot; -czf $(TarArchiveOutputPath) ."
```

Quotes are also great in `<Exec />` 😺